mlir/include/mlir/Dialect/MQTOpt/Transforms/Passes.td

```diff
@@ -103,13 +103,34 @@ def PlacementPassSC : Pass<"placement-sc", "mlir::ModuleOp"> {
              clEnumValN(PlacementStrategy::Random, "random", "Random placement"),
              clEnumValN(PlacementStrategy::Identity, "identity", "Identity placement"))}]>
   ];
+  let statistics = [
+    Statistic<"tms", "runtime-milliseconds", "The runtime in milliseconds">
+  ];
 }
 
 def RoutingPassSC : Pass<"route-sc", "mlir::ModuleOp"> {
   let summary = "This pass ensures that a program meets the connectivity constraints of a given architecture.";
   let description = [{
     This pass inserts SWAP operations to ensure two-qubit gates are executable on a given target architecture.
   }];
+  let options = [
+    Option<"method", "method", "RoutingMethod", "RoutingMethod::QMAP",
+           "The routing method to use.", [{llvm::cl::values(
+             clEnumValN(RoutingMethod::Naive, "naive", "Swap along shortest paths"),
+             clEnumValN(RoutingMethod::QMAP, "qmap", "A*-search based routing algorithm"))}]>,
+    Option<"nlookahead", "nlookahead", "std::size_t", "1",
+           "qmap option: Number of lookahead steps (heuristic horizon)">,
+    Option<"alpha", "alpha", "float", "1.0F",
+           "qmap option: The alpha factor in the cost function">,
+    Option<"beta", "beta", "float", "1.0F",
+           "qmap option: The beta factor in the cost function">,
+    Option<"lambda", "lambda", "float", "0.9F",
+           "qmap option: The lambda factor in the cost function">
+  ];
+  let statistics = [
+    Statistic<"nadd", "n-additional-gates", "The number of additional gates">,
```

Comment:
Is the `n<...>` notation established in MLIR or is this something that you came up with?
I'd have probably either written this out as `num_added` or `n_add` or `swaps_added`.
Just bike shedding though. Not the most important decision here.

Reply:
I think this is just me thinking in terms of the mathematical $$n_{add}$$. Maybe it would be smart to use `num_<>` for counting statistics and `time_<>` for timing.

Reply:
I think the added underscore already helps a lot.
So `n_add` or `n_swaps` would already work well.